build/kickstarts/base.ks.erb

```diff
@@ -259,4 +263,8 @@ rm -f /var/www/miq/vmdb/certs/v2_key
 
 [[ -s /etc/default/evm ]] && source /etc/default/evm
 appliance_console_cli --region 0 --internal --password smartvm
+
+pushd $app_root
+#copy opt dir from COPY
+cp -vr system/COPY/opt /
```

Comment:
@Fryguy 
I think we had to do that because if we copied the Postgres files first, postgres won't initialize because of the presence of the directory /opt/rh/postgresql92/root/var/lib/pgsql/data. So we have to wait till Postgres DB is initialized and then copy over the template config files EVM_V5_Shared_Appliance_postgresql.conf & EVM_V5_Dedicated_Appliance_postgresql.conf

I will add a comment in the code about why the copy was moved to the bottom after the postgres DB is initialized